admin/scripts/README.md

```diff
@@ -0,0 +1,64 @@
+# Description
+
+Automatically open issues for NLnet projects containing useful project metadata.
+
+# Setup
+
+First, make sure that you're inside the development environment:
+
+```sh
+nix-shell
+```
+
+If you have [direnv](https://github.com/nix-community/nix-direnv) installed, you can also automatically do this when you enter the directory by executing:
+
+```sh
+direnv allow
+```
+
+## Credentials
+
+Create a [fine-grained token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) with the following permissions:
+
+- Issues: Read & write
+
+Store the token in a `.env/GH_TOKEN` file. For example:
+
+```sh
+$ cat .env/GH_TOKEN
+<TOKEN>
+```
+
+## NLnet Dashboard
+
+Download projects metadata from NLnet dashboard:
+
+- Go the the [NLnet dashboard](https://dashboard.nlnet.nl) and enter your credentials
+- Go to each grant page, append `?json=true` to the URL, and download the file
+- Put all downloaded files in a single directory
+
+Then, to extract the data, run:
+
+```sh
+extract-project-metadata ./directory >>metadata.json
```

Comment:
* Change from appending to the file to always rewriting the file
* More clear example

```suggestion
extract-project-metadata <NLNET-METADATA-DIRECTORY> >metadata.json
```